tools/configuration.py

```diff
@@ -1,24 +1,40 @@
 import os
 import sys
-import getopt
+import argparse
 
 from tools import get_pipe_output
 
-default_conf = {
-    'max_domains': 10,
-    'max_ext_length': 10,
-    'style': 'gitstats.css',
-    'max_authors': 7,
-    'max_authors_of_months': 6,
-    'authors_top': 5,
-    'commit_begin': '',
-    'commit_end': 'HEAD',
-    'linear_linestats': 1,
-    'project_name': '',
-    'processes': 8,
-    'start_date': '',
-    'output': 'html'
-}
+GITSTAT_VERSION = '0.91'
```

Comment:
the version should be fetched from repostat/git_hooks/release_data.json It is automatically updated once I put a git tag of a special format and push to remote.